### PR TITLE
Fixes, Improvements, Styling

### PR DESCRIPTION
**Pull request by ryanveroniwooff**
- shortened some logic in piece.rb
- changed background images
- changed page setup and style for almost all pages (to keep things coherent)
  - obviously isn't the final draft, just something to make it look better for now
  - was really just trying to make it look centered and clean
- organized UI
- added icons for each piece and built a method to get the corresponding image for each piece
- change black board panels to wood
- added guards to not `render_pieces` or `render_pieces_selected` if there isn't anyone playing the game
This is a massive commit but it was just one little thing after another as I looked through the project. Hopefully someone is able to go through it and confirm it's all good. 
`run this in your projects directory:`
`git checkout -b improvements-and-styling origin/improvements-and-styling`
`and then run your server and lets get talking about this pull-request!`

### CODE REVIEW

File: app/models/piece.rb [hunk 1]
```diff
@@ -47,44 +47,25 @@ def capture_position(destination, destination_x, destination_y)
...
-      ((x_position + 1)...destination_x).each_with_index do |x, i|
-        y = y_position + ((i + 1) * -y_direction)
-        return true if game.pieces.exists?(x_position: x, y_position: y)
-      end
-    else
-      ((destination_x + 1)...x_position).each.with_index do |x, i|
-        y = destination_y + ((i + 1) * y_direction)
-        return true if game.pieces.exists?(x_position: x, y_position: y)
-      end
+    (x_difference < 0 ? ((x_position + 1)...destination_x) : ((destination_x + 1)...x_position)).each_with_index do |x, i|
```
Comment (jaredAnders) [inline app/models/piece.rb]: @ryanveroniwooff this and the other two obstruction methods you refactored here are awesome

File: app/views/pieces/show.html.erb [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
-
-<h3><%= @game.name %></h3>
-
-<% if user_signed_in? %>
-  <% if @game.white_player_id == nil %>
-    <%= link_to 'Join Game', game_path(@game, game: { white_player_id: current_user.id }), method: :put %>
-  <% elsif @game.black_player_id == nil && @game.white_player_id != nil %>
-    <% if @game.white_player_id == current_user.id %>
-      <p> Waiting for one more player </p>
+ <div  class="card-panel col s3 black white-text formatting center">
```
Comment (jaredAnders) [inline app/views/pieces/show.html.erb]: @mscoutermarsh do we even need this file?

Comment (ryanveroniwooff) [inline app/views/pieces/show.html.erb]: Not sure, if drag/drop is done then probably not

Comment (jaredAnders) [inline app/views/pieces/show.html.erb]: ok, we can leave it for now then

File: app/helpers/games_helper.rb [hunk 3]
```diff
@@ -1,7 +1,11 @@
 module GamesHelper
   def render_piece(x, y)
     piece = @game.pieces.find_by(x_position: x, y_position: y)
-    link_to "#{piece.type} #{piece.color}", game_piece_path(@game, piece) if piece
+    link_to image_tag(image_name(piece), class: 'responsive-img'), game_piece_path(@game, piece) if piece && @game.white_player && @game.black_player
+  end
+
+  def image_name(piece)
+    'pieces/' + piece.type + ' ' + piece.color + '.png'
```
Comment (mscoutermarsh) [inline app/helpers/games_helper.rb]: :party_parrot:

**Pull request merged**